.changeset/four-ducks-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
```

Comment:
I don't think this should change the project version at all...

Reply:
Is that possible?

Reply:
Definitely! Feel free to remove the `.changeset` here and I can add the `skip changeset` label 👍